neurom/fst/tests/test_get_features.py

```diff
@@ -66,7 +66,7 @@ def assert_items_equal(a, b):
 
 def assert_features_for_neurite(feat, neurons, expected, exact=True):
     for neurite_type, expected_values in expected.items():
-        print('neurite_type: %s' % neurite_type)
+        # print('neurite_type: %s' % neurite_type)
```

Comment:
I kept it because I think it is used for debugging the tests

Reply:
k, fair enough